llvm/lib/Transforms/Utils/SimplifyCFG.cpp

```diff
@@ -1664,18 +1664,35 @@ static bool areIdenticalUpToCommutativity(const Instruction *I1,
 static void hoistConditionalLoadsStores(
     BranchInst *BI,
     SmallVectorImpl<Instruction *> &SpeculatedConditionalLoadsStores,
-    bool Invert) {
+    std::optional<bool> Invert) {
```

Comment:
Add a comment like
```
\param Invert  ...
```
?

It's a little hard to know when it's nullopt w/o searching for the caller.

Reply:
Done.